web_client/js/views/visualization.js

```diff
@@ -58,27 +58,47 @@ histomicstk.views.Visualization = girder.View.extend({
             }, this));
         });
 
-        this.listenTo(histomicstk.events, 'query:image', _.bind(function (image) {
+        this.listenTo(histomicstk.events, 'query:image', function (image) {
             var currentImage = this._controlModel.get('value') || {};
             if (image && currentImage.id !== image) {
                 this._controlModel.set('value', new girder.models.ItemModel({_id: image}));
             } else if (!image) {
                 this.removeItem();
                 this._controlModel.set('value', null);
             }
-        }, this));
+        });
+
+        // Set image bounds on URL query parameter change
+        this.listenTo(histomicstk.events, 'query:bounds', this._boundsFromQuery);
 
         // fallback to canvas renderer rather than dom
         geo.gl.vglRenderer.fallback = function () {return 'canvas';};
     },
 
     /**
+     * Set the displayed image bounds according to the given query string.
+     */
+    _boundsFromQuery: function (query) {
+        if (!this._map || !query) {
+            return;
+        }
+        var bounds = query.split(',').map(function (v) { return +v; });
+        this._map.rotation(bounds[4] * Math.PI / 180);
+        this._map.bounds({
+            left: bounds[0],
+            top: bounds[1],
+            right: bounds[2],
+            bottom: bounds[3]
+        }, null);
```

Comment:
By passing `null` as a second parameter here and in the `var bounds = this._map.bounds(undefined, null);` call below, we are in the map gcs coordinates, rather than the interface coordinate system.  The difference is that the interface coordinate system is 0,0 is left, top and positive values are right, bottom, while the map gcs is right-handed with 0,0 left, top, with positive values to the right and *negative* values to the left.  Just removing the `null` (not padding a second parameter or passing undefined), we can stay with the interface coordinate system and have the coordinates in image pixels.